.github/workflows/build-user-config.yml

```diff
@@ -52,20 +52,37 @@ jobs:
       fail-fast: false
       matrix: ${{ fromJson(needs.matrix.outputs.build_matrix) }}
     steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Create build directory
+        run: |
+          echo "build_dir=$(mktemp -d)" >> $GITHUB_ENV
+
       - name: Prepare variables
         shell: sh -x {0}
         env:
           board: ${{ matrix.board }}
           shield: ${{ matrix.shield }}
           artifact_name: ${{ matrix.artifact-name }}
         run: |
+          export new_tmp_dir=$(mktemp -d)
+          echo "tmp_dir=${new_tmp_dir}" >> $GITHUB_ENV
+          echo "tmp_config_dir=${new_tmp_dir}/config" >> $GITHUB_ENV
+          if [ -e zephyr/module.yml ]; then
+            export zmk_load_arg=" -DZMK_EXTRA_MODULES='${GITHUB_WORKSPACE}'"
+          fi
+
+          echo "load_arg=${zmk_load_arg}" >> $GITHUB_ENV
```

Comment:
I don't think this line is doing anything, is it?

Reply:
Nope! Removing.